lib/features/family/features/reflect/domain/grateful_recommendations_repository_impl.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:givt_app/features/family/features/recommendation/organisations/models/organisation.dart';
+import 'package:givt_app/features/family/features/recommendation/organisations/repositories/organisations_repository.dart';
+import 'package:givt_app/features/family/features/reflect/domain/grateful_recommendations_repository.dart';
+import 'package:givt_app/features/family/features/reflect/domain/models/game_profile.dart';
+
+class GratefulRecommendationsRepositoryImpl
+    implements GratefulRecommendationsRepository {
+
+  GratefulRecommendationsRepositoryImpl(this._organisationsRepository);
+
+  final OrganisationsRepository _organisationsRepository;
+  final Map<GameProfile, List<Organisation>> _gratefulRecommendations = {};
+
+  @override
+  Future<Map<GameProfile, List<Organisation>>>
```

Comment:
This method is going to fetch and store the data in the repository I think? :) 
Should it have a return type or just void?

Reply:
Yes. Could also be void but then I would rename it.